### ISSUE TITLE
Main wiki page linking to potential malware

### ISSUE THREAD

**Issue by Ark-kun** [rclone/rclone#6834]
I understand it's Wiki, but maybe it should be more locked down.

The change added the "Rclone for Windows" button to the main Wiki page and many other pages. The button pointed to some suspicious-looking location.

I've rolled back the changes.

I understand it's Wiki, but maybe it should be more locked down.

https://github.com/rclone/rclone/wiki/Home/_compare/31a1c97d8a897e00101094541ae2004a6a7e9223...d45831e81bfc046310bfc68ad8514a71ce3f6cdf#diff-a5d27d3be3c0c9e75920dc813042418e300bc4a99eb4f7ab0433139244b43bc8

**Comment by ncw** [comment]
Thanks for reverting the change.

I guess we could lock it down to collaborators - but that harms the wiki if you need to be a project member to add anything.

Ideas @rclone/rclone ?

**Comment by darthShadow** [comment]
Do we really need a wiki separately from the rclone website? Going through it, I don't see a reason it needs to be separate rather than a subsection on the website itself that can be modified via pull requests with looser restrictions about the formatting and content.

**Comment by animosity22** [comment]
It's a shame the human race sucks.

Probably has to be locked down.

**Comment by albertony** [comment]
> Do we really need a wiki separately from the rclone website? Going through it, I don't see a reason it needs to be separate rather than a subsection on the website itself that can be modified via pull requests with looser restrictions about the formatting and content.

I tend to agree with @darthShadow. We already have the website and the forum as primary sources of information. The wiki I think is more difficult to find, and my impression is that new users don't go there without being told to from a forum post (I have no idea if this is true or not, just my impression). If we were to lock down the wiki, moving it to the docs/website would not make it that much different - as long as it is in a section for user contributions with looser restrictions - and a disclaimer that this is what it is. Non-members that don't feel confident supplying a pull request can always write a how-to in the forum, and it can be added to the docs later.
